lookout/style/format/analyzer.py

```diff
@@ -177,29 +162,43 @@ def _files_by_language(cls, files: Iterable[File]) -> Dict[str, Dict[str, File]]
 
     @classmethod
     def _load_config(cls, config: Mapping[str, Any]):
-        final_config = {
-            "siblings_window": 5,
-            "parents_depth": 2,
-            "lower_bound_instances": 500,
-            "prune_branches_algorithms": ["reduced-error"],
-            "top_down_greedy_budget": [False, .5],
-            "prune_attributes": False,
-            "uncertain_attributes": True,
-            "prune_dataset_ratio": .2,
-            "n_estimators": 10,
-            "random_state": 42,
-            "n_iter": 5,
-            "debug_parsing": False,
-            "line_length_limit": 500
+        def merge(default_config, config):
```

Comment:
Please rename the function `recursive_update` and the arguments `mapping` and `other` to follow standard lib a bit more